cub/benchmarks/bench/transform/common.h

```diff
@@ -25,9 +25,10 @@
 
 #include <nvbench_helper.cuh>
 
-template <typename... RandomAccessIteratorsIn>
+template <typename RandomAccessIteratorOut, typename... RandomAccessIteratorsIn>
 #if TUNE_BASE
-using policy_hub_t = cub::detail::transform::policy_hub<false, ::cuda::std::tuple<RandomAccessIteratorsIn...>>;
+using policy_hub_t =
+  cub::detail::transform::policy_hub<false, ::cuda::std::tuple<RandomAccessIteratorsIn...>, RandomAccessIteratorOut>;
```

Comment:
important: policy hub below doesn't support vectorized algorithm yet surrounding benchmarks have:
```
// %RANGE% TUNE_ALGORITHM alg 0:1:1
```
Now the definition of `1` changed to vectorized, so tuning will lead to algorithm = vectorized and algo_policy = async. Let's fix this before merging. After fix, please verify that tuning of transform works.

Reply:
I would like to address tuning of the vectorized version after this PR and #2394, #4976, + my internal rewrite of memcpy_async lands. There is no point in tuning before all code paths and optimizations are in.

Reply:
#5057